packages/react-query/src/createTRPCReact.tsx

```diff
@@ -90,15 +90,6 @@ export type DecorateProcedure<
   TPath extends string,
 > = TProcedure extends AnyQueryProcedure
   ? {
-      /**
-       * Method to extract the query key for a procedure
-       * @param type - defaults to `any`
-       * @link https://trpc.io/docs/useContext#-the-function-i-want-isnt-here
-       */
-      getQueryKey: (
-        input: inferProcedureInput<TProcedure>,
-        type?: QueryType,
-      ) => QueryKey;
```

Comment:
Restore this and mark it deprecated - we'll remove it on `next`:

```suggestion
      /**
        * Method to extract the query key for a procedure
        * @param type - defaults to `any`
        * @link https://trpc.io/docs/useContext#-the-function-i-want-isnt-here
        * @deprecated
        */
       getQueryKey: (
         input: inferProcedureInput<TProcedure>,
         type?: QueryType,
       ) => QueryKey;
```